client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -152,27 +152,45 @@ class RegisteredDomainType extends React.Component {
 		const { expiry } = domain;
 
 		if ( isExpiringSoon( domain, 30 ) ) {
+			let message;
+			if ( domain.currentUserCanManage ) {
+				message = translate(
+					'Your domain will expire in {{strong}}%(days)s{{/strong}}. Please renew it before it expires or it will stop working.',
+					{
+						components: {
+							strong: <strong />,
+						},
+						args: {
+							days: moment.utc( expiry ).fromNow( true ),
+						},
+					}
+				);
+			} else {
+				message = translate(
+					'The domain will expire in {{strong}}%(days)s{{/strong}}. Please contact the domain owner (%(owner)s) to renew it.',
```

Comment:
Looking at some other places we do this. We can drop the parenthesis around the owner's name

**Before**
> The domain has expired and is no longer active. Please contact the domain owner (Filippo Di Trapani (fditrapani)) to restore it.

**After**
> The domain has expired and is no longer active. Please contact the domain owner **Filippo Di Trapani (fditrapani)** to restore it.